Sources/Client/Client.h

```diff
@@ -395,6 +396,7 @@ namespace spades {
 			virtual ~Client();
 
 		public:
+			FPSCounter upsCounter;
```

Comment:
Don’t add it as a public member, please. I suggest to move it to `NetClient` and then expose the ups value via a public method.

Reply:
Yes, that was the original plan. But then I got an error about it being incomplete (NetClient and Client would have a cyclic dependency?) and gave up because I basically only quickly wanted this information once.

I'd probably have to brush up on my C++ to figure out how to deal with that.

Reply:
Would adding a MarkWorldUpdate() be better? That's the other idea I had to bring this up to OpenSpades code quality.

Reply:
Yep, that’d be better.